ui/app/routes/allocations/allocation.js

```diff
@@ -48,5 +48,5 @@ export default class AllocationRoute extends Route.extend(WithWatchers) {
   @watchRecord('allocation') watch;
   @watchNonStoreRecords('allocation') watchHealthChecks;
 
-  @collect('watch') watchers;
+  @collect('watch', 'watchHealthChecks') watchers;
```

Comment:
Does this work because `watchers` are subject to [visibilityHandler()](https://github.com/hashicorp/nomad/blob/main/ui/app/mixins/with-watchers.js#L28-L34) ?